src/main/java/com/epam/esm/model/Certificate.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.esm.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Set;
+
+public class Certificate implements Serializable {
+
+   private Long id;
```

Comment:
Move id field to abstract class (for Tag as well )

Reply:
Done